src/java.base/share/classes/java/lang/System.java

```diff
@@ -813,6 +814,11 @@ public static native void arraycopy(Object src,  int  srcPos,
      * Note that even if the security manager does not permit the
      * {@code getProperties} operation, it may choose to permit the
      * {@link #getProperty(String)} operation.
+     * <p>
+     * In addition to the above set of system properties, locale related
```

Comment:
I think I'd consider a slight reduction in the first/last phrases leaving this equivalent statement:

Additional locale-related system properties defined by the {@link Locale##default_locale Default Locale} section in the {@code Locale} class description may also be obtained with this method.

(Note that I couldn't find the referenced Locale section, but perhaps I was looking at an older version of the spec.)

Reply:
That's concise and clearer. Replaced the wording. As to the referenced Locale section, it is in the PR (#17065) which this PR is dependent on and is not yet integrated. Sorry for the confusion.